frontend/integration-tests/protractor.conf.ts

```diff
@@ -36,7 +36,6 @@ export const config: Config = {
     chromeOptions: {
       args: [
         '--disable-gpu',
-        '--headless',
```

Comment:
I'm guessing this just fell through the cracks, but we probably don't want to remove this arg.

Reply:
it's  by a mistake, add the line back.